gantry/util/gitlab.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import aiohttp
+
+
+class GitlabClient:
+    def __init__(self):
+        self.base_url = os.environ["GITLAB_URL"]
+        self.headers = {"PRIVATE-TOKEN": os.environ["GITLAB_API_TOKEN"]}
```

Comment:
```suggestion
    def __init__(self, base_url: str, api_token: str):
        self.base_url = base_url
        self.headers = {"PRIVATE-TOKEN": api_token}
```

I'd suggest instantiating the class with the config options rather than getting them from `os.environ` that way we can use this code a true library instead of having it be tried directly to gantry's config.